.ddev/commands/web/lint

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+## Description: Run lint
+## Usage: lint
+## Example: "ddev lint ci:ts:lint"
+
+composer $1
```

Comment:
As this command executes `composer` within the container (and it's not limited to linting), I propose we rename it to * composer`.

Reply:
I wanted to create separate commands for tests etc. but I will remove this command, because ddev is providing ddev composer args by default.